### PR TITLE
[ISSUE 748] documentation template

### LINKED ISSUE #748: Documentation template
**Is your feature request related to a problem? Please describe.**
Currently we have template for proposals - https://github.com/dell/csi-baremetal/blob/master/docs/proposals/TEMPLATE.md but its format doesn't fit well to the final feature documentation.

**Describe the solution you'd like**
Need to have template for final documentation

**Describe alternatives you've considered**
None

**Additional context**
None

### PR DESCRIPTION
**Pull request by mishoyama**
### Resolves #748 

Documentation template to use for new articles or proposal conversions.

## PR checklist
- [x] Add link to the issue
- [x] Choose Project
- [x] Choose PR label
- [x] All comments are resolved

### CODE REVIEW

File: docs/DOC_TEMPLATE.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ versions:
 <!-- Great intros give readers a quick understanding of what's in the article, so they can tell whether it's relevant to them before moving ahead. For more tips, see https://github.com/github/docs/blob/main/contributing/content-model.md#intro. -->
 <!-- For product callout info, see https://github.com/github/docs/tree/main/content#product.-->
 <!-- For product version instructions, see https://github.com/github/docs/tree/main/content#versions.-->
+<!-- If you need to draw a diagram, use https://app.diagrams.net/. Don't forget to save it in xlm format as well for edit option.-->
```
Comment (rkiselenko) [inline docs/DOC_TEMPLATE.md:16]: Export as `svg` and embed your diagram to md files, this file can be imported to https://app.diagrams.net again to add any changes. `svg` file also editable in https://app.diagrams.net

Comment (mishoyama) [reply docs/DOC_TEMPLATE.md]: Updated

**Review by misanthropicat — APPROVED** [review]

File: docs/proposals/TEMPLATE.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 [This is a template for CSI-Baremetal's changes proposal. ]
+<!-- If you need to draw a diagram, use https://app.diagrams.net/. Don't forget to save it in xlm format as well for edit option.-->
```
Comment (rkiselenko) [inline docs/proposals/TEMPLATE.md:2]: This note must be changed too, no need xml.

**Review by rkiselenko — CHANGES_REQUESTED** [review]

File: docs/proposals/TEMPLATE.md (hunk 2 above)
Comment (mishoyama) [reply docs/proposals/TEMPLATE.md]: Done

**Review by rkiselenko — APPROVED** [review]

**Pull request merged**